javaslang/src/main/java/javaslang/Predicates.java

```diff
@@ -118,4 +119,23 @@ private Predicates() {
         Objects.requireNonNull(predicates, "predicates is null");
         return anyOf(predicates).negate();
     }
+
+
+    /**
+     * Creates a {@code Predicate} that tests, if an object is null
+     * @param <T> value type
+     * @return A new {@code Predicate}
+     */
+    public static <T> Predicate<T> isNull() {
+        return Objects::isNull;
+    }
+
+    /**
+     * Creates a {@code Predicate} that tests, if an object is not null
+     * @param <T> value type
+     * @return A new {@code Predicate}
+     */
+    public static <T> Predicate<T> isNotNull() {
+        return Objects::nonNull;
+    }
```

Comment:
Thank you Grzegorz.
What is your opinion about the names isNotNull vs. nonNull?
I like that isNotNull has the prefix 'is', which is typical for predicates. Personally I'm used to it.
I like about nonNull that it is a Java standard and that it is shorter. Personally I'm not used to it.
Once we defined the predicates, they should stay untouched to keep the API stable.